samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb

```diff
@@ -4,7 +4,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# FfDL Kubeflow Pipeline Notebook demo\n"
+    "# Train and deploy with FfDL and Seldon demo\n"
+   ]
+  },
+  {
+   "attachments": {
+    "Screen%20Shot%202019-02-15%20at%204.50.17%20PM.png": {
```

Comment:
Is it possible to link to the images?

Reply:
Would love to have the notebook as a unit which somebody can just take, without reference to anything out of the notebook

Reply:
@gaoning777 @hongye-sun @IronPan @vicaire WDYT?

Reply:
Hmm. GitHub does not seem to show the images:

![image](https://user-images.githubusercontent.com/1829149/56829067-9d0e3000-6817-11e9-97f9-e51c1c142ea4.png)

Also TF seems to be using HTML IMG even for small images https://github.com/tensorflow/docs/blob/b5cb76b01b96fb936edb29ac0323a2d630d575ac/site/en/r2/tutorials/eager/tf_function.ipynb

Reply:
I would vote for a link in the ipynb file and publish the image in the same directory or on the internet. 
1) I've searched throughout the web about embedding images in ipynb, people always attach a link to the image in the ipynb. 
2) Sharing a notebook vs. a directory does seem to be much different. Plus, the image can always be published in a publicly available cloud storage if the author wants to. 
3) It does not seem to be rendered correctly on the github.

Reply:
Ok removed the image reference for now. Will revisit later